Starcounter.Uniform/FormItem/FormItemMessage.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Starcounter.Uniform.FormItem
+{
+    /// <summary>
+    /// Class that represents message object of a form item.
+    /// </summary>
+    public class FormItemMessage
+    {
+        /// <summary>
+        /// Message text.
+        /// </summary>
+        public string Text { get; set; }
+
+        /// <summary>
+        /// Can be true, false or empty which represents invalid, valid and neutral message.
+        /// </summary>
+        public string Type { get; set; }
```

Comment:
can we use Enum here?

Reply:
I would stick with string, since this is the base returned type and may ease potential debugging since we skip another translation from string to enum.

Reply:
> base returned type
what does that mean?
> may ease potential debugging
that's good, but our priority should be the regular use of our code, not debugging. The user can set message type with enum, so they should be able to retrieve it as enum